src/Blazored.LocalStorage/Blazored.LocalStorage.csproj

```diff
@@ -6,7 +6,7 @@
     <RootNamespace>Blazored.LocalStorage</RootNamespace>
 
     <PackageId>Blazored.LocalStorage</PackageId>
-    <Version>2.1.5</Version>
+    <Version>2.1.6-preview-0.0.2</Version>
```

Comment:
Could you undo this one? We set the build number when this goes to release.

Reply:
No problems :-)